apis/ec2/src/main/java/org/jclouds/ec2/compute/domain/EC2HardwareBuilder.java

```diff
@@ -416,6 +431,89 @@ public static EC2HardwareBuilder t2_large() {
             .processors(ImmutableList.of(new Processor(1.0, 0.4))).rootDeviceType(RootDeviceType.EBS);
    }
 
+   /**
+    * @see InstanceType#T2_XLARGE
+    */
+   public static EC2HardwareBuilder t2_xlarge() {
+      return new EC2HardwareBuilder(InstanceType.T2_XLARGE).t2()
+              .ram(16384)
+              .processors(ImmutableList.of(new Processor(2.0, 0.4))).rootDeviceType(RootDeviceType.EBS);
```

Comment:
assuming the first param of the Processor constructor is the number of vCPUs, then this should be 4, not 2 for a t2.xlarge

Reply:
good catch, yup ```

t2.xlarge | 4 | 16
-- | -- | --
t2.2xlarge | 8 | 32
```